### PR TITLE
ACCUMULO-4630 Move user manual to Accumulo website

### PR DESCRIPTION
**Pull request by mikewalch**
* removed user manual asciidoc and moved it to accumulo-website repo
* binary tarball no longer contains user manual
* moved configuration mgmt docs in 'config-header.txt' to website
* ConfigurationDocGen.java now generates markdown for website during
  core module build to core/target/generated-docs

See https://github.com/apache/accumulo-website/pull/11 for corresponding PR to add documentation to website.

### CODE REVIEW

File: core/src/main/java/org/apache/accumulo/core/conf/ConfigurationDocGen.java [hunk 1]
```diff
@@ -37,174 +31,115 @@
...
 
     @Override
     void prefixSection(Property prefix) {
       boolean depr = prefix.isDeprecated();
-      doc.println("[[" + prefix.name() + "]]");
-      doc.println("==== " + prefix.getKey() + "*" + (depr ? " (Deprecated)" : ""));
-      doc.println(strike((depr ? "_Deprecated._ " : "") + sanitize(prefix.getDescription()), depr));
-      doc.println();
+      doc.print("| <a name=\"" + prefix.getKey().replace(".", "_") + "prefix\"></a> **" + prefix.getKey() + "*** | ");
+      doc.println((depr ? "**Deprecated.** " : "") + strike(sanitize(prefix.getDescription()), depr) + " |");
```
Comment (ctubbsii) [inline core/src/main/java/org/apache/accumulo/core/conf/ConfigurationDocGen.java]: Could also make the `**` a `bold(String)`

File: core/src/main/java/org/apache/accumulo/core/conf/Property.java [hunk 2]
```diff
@@ -86,7 +86,7 @@
           + "the configuration file, set this property to true."),
   // SSL properties local to each node (see also instance.ssl.enabled which must be consistent across all nodes in an instance)
   RPC_PREFIX("rpc.", null, PropertyType.PREFIX, "Properties in this category related to the configuration of SSL keys for RPC. See also instance.ssl.enabled"),
-  RPC_SSL_KEYSTORE_PATH("rpc.javax.net.ssl.keyStore", "", PropertyType.PATH, "Path of the keystore file for the servers' priva:q" + "te SSL key"),
```
Comment (ctubbsii) [inline core/src/main/java/org/apache/accumulo/core/conf/Property.java]: Heh, `vi`-based typos. :)

File: pom.xml [hunk 3]
```diff
@@ -257,13 +256,6 @@
       </dependency>
       <dependency>
         <groupId>org.apache.accumulo</groupId>
-        <artifactId>accumulo-docs</artifactId>
```
Comment (keith-turner) [inline pom.xml]: There is still asciidoc plugin info in this pom, should that be removed?

File: core/pom.xml [hunk 4]
```diff
@@ -259,6 +259,41 @@
...
+            <goals>
+              <goal>java</goal>
+            </goals>
+            <phase>compile</phase>
+            <configuration>
+              <mainClass>org.apache.accumulo.core.conf.ConfigurationDocGen</mainClass>
+              <classpathScope>compile</classpathScope>
+              <arguments>
+                <argument>--generate-markdown</argument>
+                <argument>${project.build.directory}/generated-docs/configuration-properties.md</argument>
```
Comment (keith-turner) [inline core/pom.xml]: Will this be used by the website docs?

Comment (mikewalch) [inline core/pom.xml]: Yes, we still need to generate documentation for the configuration properties and copy it to the accumulo website repo.

File: pom.xml (hunk 3 above)
Comment (mikewalch) [inline pom.xml]: It should be removed.  I will push another commit.

**Comment by ctubbsii** [comment]
The referenced JIRA issue remains open. Is there more work to be done?

**Comment by mikewalch** [comment]
I closed it

**Pull request merged**